src/client/src/rt-theme/themes.ts

```diff
@@ -71,25 +73,27 @@ export interface ColorPair {
   textColor?: string
 }
 
-export const themes = {
-  light: createTheme(colors.light, colors.accents),
-  dark: createTheme(colors.dark, colors.accents, (theme: Theme) => ({
-    button: {
-      ...theme.button,
-      secondary: {
-        ...theme.button.secondary,
-        textColor: theme.primary.base
-      }
+const darkGenerated: ReturnType<typeof generateTheme> = generateTheme(colors.dark, colors.accents)
+
+const darkModified = {
```

Comment:
This seems messy and indirect — we're changing the solution for the tool.

I'd revert.

Reply:
Updated. Keeps the original interface with typing intact